packages/cyberstorm/README.md

```diff
@@ -0,0 +1,74 @@
+# Cyberstorm UI library
+
+## Storybook
+
+[Storybook](https://storybook.js.org/docs/react/get-started/introduction)
+provides a sandbox to build UI components in isolation, without having to start
+up the whole service stack. Additionally it showcases the existing components,
+promoting reusability.
+
+To start Cyberstorm Storybook, run `yarn workspace @thunderstore/cyberstorm-storybook storybook`.
+
+Storybook can then be accessed at [http://localhost:6006/].
```

Comment:
This link won't render correctly (see rich diff). Not a blocker for merge IMO but mentioning either way.

Reply:
A bit better now.